<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroDialog</class>
 <widget class="QDialog" name="IntroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Widget_bk_small" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_bk_small" stretch="0,0,0,0,0,0">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="storageLabel">
        <property name="text">
         <string>由于这是第一次启动此程序，您可以选择 %1 的数据所存储的位置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dataDirDefault">
        <property name="text">
         <string>使用默认的数据目录(推荐)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dataDirCustom">
        <property name="text">
         <string>使用自定义的数据目录：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="dataDirectory"/>
          </item>
          <item>
           <widget class="QPushButton" name="ellipsisButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="freeSpace">
          <property name="text">
           <string>(大约需要 60GB 空间)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="errorMessage">
          <property name="text">
           <string>一个新的数据目录将被创建</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxCopy">
        <property name="text">
         <string>是否拷贝钱包数据库(推荐拷贝)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IntroDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IntroDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
